src/LayoutComponents/nav/NavPanel.jsx

```diff
@@ -109,13 +109,16 @@ class NavPanel extends PureComponent {
   }
 
   render() {
-    const { isExpanded } = this.props;
+    const { isExpanded, title } = this.props;
+    const panelId = `${title.toLowerCase().replace(/\s/g, '-')}-panel`;
```

Comment:
If you explore `category` in `mapstateToProps` for this component, you will find a prop called `dashedName` which is what you are creating here, minus the `-panel` bit.